packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/getDisplayedFilters.js

```diff
@@ -0,0 +1,90 @@
+import ComboboxFilter from '../ComboboxFilter';
+import { getDefaultMessage, actionTypes } from './formatMessage';
+
+const customOperators = [
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$eq', defaultMessage: 'is' },
+    value: '$eq',
+  },
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$ne', defaultMessage: 'is not' },
+    value: '$ne',
+  },
+];
+
+const getDisplayedFilters = ({ formatMessage, users }) => {
+  const actionOptions = Object.keys(actionTypes).map((action) => {
+    return {
+      label: formatMessage({
+        id: `Settings.permissions.auditLogs.${action}`,
+        defaultMessage: getDefaultMessage(action),
+      }),
+      customValue: action,
+    };
+  });
+
+  const userOptions = users?.results.map((user) => {
+    return {
+      label: `${user.firstname} ${user.lastname}`,
```

Comment:
Same here. Can we move this into a translation string, because the order might not be the same in every language.